c10/core/impl/CUDATrace.cpp

```diff
@@ -0,0 +1,22 @@
+#include <mutex>
+
+#include <c10/core/impl/CUDATrace.h>
+#include <c10/util/CallOnce.h>
+
+namespace c10 {
+namespace impl {
+
+std::atomic<const PyInterpreter*> CUDATrace::cudaTraceState;
```

Comment:
Nit: explicitly initialize it to nullptr? I think this is already the case, since static globals are zero-initialized, but better be safe than sorry? ;)

Reply:
I'm pretty sure it is, but I've changed it to be extra-safe.